content/site-map.yaml

```diff
@@ -55,6 +55,8 @@
               file: "/events/ashg2023-american-society-of-human-genetics"
             - name: ""
               file: "/events/ashg2023-satisfying-nih-dms-requirements-with-anvil"
```

Comment:
@esthermcdade would you mind ordering the event `Satisfying NIH data management and sharing (DMS) requirements with AnVIL` below AnVIL Demo for September and October? Thank you.